operators/build.sh

```diff
@@ -42,11 +42,6 @@ directories=(
   histograms
 )
 
-if [ "$with_vis" == true ]; then
-  directories+='stateful_volume_renderer'
-  directories+='vtk_reader'
-fi
-
```

Comment:
We can probably remove the `with_vis` variable above as well.

Reply:
Sure, will go that

Reply:
Done